components/server/src/lib.rs

```diff
@@ -7,5 +7,6 @@ extern crate tikv_util;
 pub mod setup;
 pub mod memory;
 pub mod raft_engine_switch;
+pub mod raftstore_v1;
```

Comment:
It's very strange to have a raftstore module in server.

Reply:
it's meant to put raftstore_v1 related implementation.  For now there's only one struct, the submodule seems an overkill. 
I don't like to see too many raftstore v1 and raftstore v2 implementation.  But with today v2's interface change,  perhaps we have to do this.

Reply:
Perhaps we should move these implementation to raftstore component instead.

Reply:
good point

Reply:
It has a problem in raftstore----raftstore is supposed to be KvEngine agnostic, but here we're dealing with specific RockDBEngine.